### PR TITLE
Limits rating range (0-5) in the API

### LINKED ISSUE #106: Limit star rating to a 0-5 range
Both in the API and frontend.

See #102

### PR DESCRIPTION
**Pull request by hermzz**
Fixes #106

### CODE REVIEW

File: api/v1/recipe/serializers.py [hunk 1]
```diff
@@ -59,6 +60,13 @@ def update(self, instance, validated_data):
         direction_data = validated_data.pop('directions', None)
         tag_data = validated_data.pop('tags', None)
 
+        rating = int(validated_data.get('rating', 0))
+        if rating < 0:
+            rating = 0;
```
Comment (RyanNoelk) [inline api/v1/recipe/serializers.py]: semi-colon :P

**Review by RyanNoelk — COMMENTED** [review]
small nit, otherwise LGTM. Also +1 for tests!

File: api/v1/recipe/serializers.py (hunk 1 above)
Comment (hermzz) [reply api/v1/recipe/serializers.py]: Hah, whoops!

**Review by RyanNoelk — APPROVED** [review]
LGTM

**Pull request merged**